src/node.cc

```diff
@@ -909,6 +887,34 @@ static ExitCode InitializeNodeWithArgsInternal(
     if (exit_code != ExitCode::kNoFailure) return exit_code;
   }
 
+  if (!per_process::cli_options->per_isolate->per_env->env_file.empty()) {
```

Comment:
I'm not sure how to add a test for this but:

You need to add a test where node is built with `NODE_WITHOUT_NODE_OPTIONS` and we don't set NODE_OPTIONS.